test/evp_test.c

```diff
@@ -2290,6 +2288,7 @@ static int encapsulate(EVP_TEST *t, EVP_PKEY_CTX *ctx, const char *op,
     *outsecret = secret;
     *outwrappedlen = wrappedlen;
     *outsecretlen = secretlen;
+ok:
```

Comment:
I'd get rid of the `end:` label, and the `goto end;`:
```suggestion
ok:
     ret = 1;
err:
    if (ret == 0) {
        OPENSSL_free(wrapped);
        OPENSSL_free(secret);
    }         
```

Reply:
Done